templates/flavors/k3s/full-vpcless/kustomization.yaml

```diff
@@ -178,3 +179,19 @@ patches:
     patch: |-
       - op: remove
         path: /spec/template/spec/firewallRef
+  - target:
+      group: infrastructure.cluster.x-k8s.io
+      version: v1alpha2
+      kind: LinodeMachineTemplate
+    patch: |-
+      apiVersion: infrastructure.cluster.x-k8s.io/v1alpha2
+      kind: LinodePlacementGroup
```

Comment:
question: Is this suppose to be kind: LinodePlacementGroup? Wouldn't this be LinodeMachineTemplate?

Reply:
Yup, that should be LinodeMachineTemplate! Changing it